uCNC/mcus/README.md

```diff
@@ -3,9 +3,9 @@
 
 ## How to add a custom MCU to µCNC
 µCNC can be extended to other custom MCU as well. For that you only need to create two files:
-  1. A correspondent mcumap.h file that defines the pins equivalent masks in your MCU and some custom definitions to access ROM/Flash memory and special math operations (check [mcumap_virtual.h](https://github.com/Paciente8159/uCNC/blob/master/uCNC/mcus/virtual/mcumap_virtual.h) and [mcumap_atmega328p.h](https://github.com/Paciente8159/uCNC/blob/master/uCNC/mcus/avr/uno/mcumap_atmega328p.h) to get an idea).
-  2. A file that implements all the functions defined in the [mcu.h](https://github.com/Paciente8159/uCNC/blob/master/uCNC/mcu.h) header file. mcu.h already has implemented the necessary instructions to perform direct IO operations to digital pins.
+  1. A correspondent `mcumap_YOUR-MCU.h` file that defines the pins equivalent masks in your MCU, some custom definitions to access ROM/Flash memory, and special math operations (check [mcumap_virtual.h](https://github.com/Paciente8159/uCNC/blob/master/uCNC/mcus/virtual/mcumap_virtual.h) and [mcumap_atmega328p.h](https://github.com/Paciente8159/uCNC/blob/master/uCNC/mcus/avr/uno/mcumap_atmega328p.h) to get an idea).
```

Comment:
Actually the architecture is something like this:
![mcu_custom](https://user-images.githubusercontent.com/12116536/73069005-84b16b80-3ea4-11ea-8cd3-272978d638b5.png)
I will add this schematic to the project later.